defaults/main.yml

```diff
@@ -9,6 +9,7 @@ mobilizon_identifier: mobilizon
 mobilizon_uid: ''
 mobilizon_gid: ''
 
+# renovate: datasource=docker depName=framasoft/mobilizon
```

Comment:
Please note that the official images remain hosted at https://hub.docker.com/r/framasoft/mobilizon.